components/switch/test/switch.test.js

```diff
@@ -9,9 +9,10 @@ describe('d2l-switch', () => {
 		runConstructor('d2l-switch');
 	});
 
-	it('dispatches change event', async() => {
+	it.only('dispatches change event', async() => {
```

Comment:
I've done this so many times lol...

```suggestion
	it('dispatches change event', async() => {
```

Reply:
Nice catch! 😮‍💨